transport-native-io_uring/src/main/c/netty_io_uring_native.c

```diff
@@ -264,8 +264,13 @@ static jlongArray netty_io_uring_setup(JNIEnv *env, jclass clazz, jint entries)
     memset(&p, 0, sizeof(p));
 
 #ifdef IORING_SETUP_SUBMIT_ALL
-    p.flags = IORING_SETUP_SUBMIT_ALL;
+    p.flags |= IORING_SETUP_SUBMIT_ALL;
 #endif
+    p.flags |= IORING_SETUP_R_DISABLED;
+
+    // TODO: Check if these are supported
+    p.flags |= IORING_SETUP_SINGLE_ISSUER;
+    p.flags |= IORING_SETUP_DEFER_TASKRUN;
```

Comment:
This flag is the one that makes the huge difference.... The others are just a requirement for this one to work for us.